modules/ROOT/pages/as2/as2-connectorMule4.adoc

```diff
@@ -0,0 +1,187 @@
+|Requester config - MDN MIC Signature Algorithm| The algorithm with which MIC is calculated for the MDN/receipt signature. `UNSIGNED` by default, that is, message is not signed.
+|Requester config - Request a receipt| Option to request a receipt from the partner. `SIGNED_OPTIONAL` by default.
+|Requester config - Request delivery URL| Asynchronous receipt delivery URL which points to the MDN listener end point.
+|Requester config - Encryption Algorithm| The encryption algorithm for encrypting the message. `UNENCRYPTED` by default.
+|Requester config - MIC Signature Algorithm| The algorithm with which MIC is calculated for the message signature. `UNSIGNED` by default, that is, message is not signed.
+|Requester config - Content transfer encoding| The content transfer encoding. `BINARY` by default.
+|Requester config - Compression type| The compression type. ZLIB is the only supported compression/decompression. `NONE` by default.
+|Keystore config - Keystore Path| The path of the keystore where certificates and keys are provided.
+|Keystore config - Keystore Password| The password for the keystore.
+|Keystore config - Private key Password| If private key requires a password.
+|===
++
+. Configure a connection to the partner AS2 endpoint by providing the `Partner Base URL`.
+. If you want to configure other connection details such as TLS, proxy, timeouts you can do that now.
+. Choose to send the file/message by either using `Send with sync mdn` or `Send with async mdn`:
+
+** The payload is the message content to be sent.
+
+** Filename is the file name of the message being sent.
+
+** Boolean to configure if the process should fail or if the receipt status is not processed.
+
+NOTE: If you choose to request an asynchronous receipt, you need to configure `MDN Listener`.
+
+=== Configure a MDN Listener
+
+Place the connector’s `MDN Listener` as a source in your flow as applicable for your use case and reference the global config defined here.
+
+[%header%autowidth.spread]
+|===
+|Key| Description
+|Self config - AS2 OID| Sender partner’s OID.
+|Self config - x509 alias| Sender’s Certificate and/or private key alias in the keystore.
+|Self config - Email| Sender partner’s email.
+|Partner config - AS2 ID| Receiver partner’s OID.
+|Partner config - x509 alias| Receiver’s Certificate and/or private key alias in the keystore.
+|Partner config - Email| Receiver partner’s email.
+|HTTP Listener config| Reference to HTTP Listener global config.
+|Keystore config - Keystore Path| The path of the keystore where certificates and keys are provided.
+|Keystore config - Keystore Password| The password for the keystore.
+|Keystore config - Private key Password| If the private key requires a password.
+|===
+
+=== Configure an AS2 Listener for receiving a message
```

Comment:
```suggestion
=== Configure an AS2 Listener for Receiving a Message
```

Reply:
Should this be a peer example: "==Example: Configure a Receive AS2 Message"?
That would answer the edit question below....